cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -279,6 +282,16 @@ share
     txId                TxId
     UniqueTreasury      addrId txId
 
+  -- -----------------------------------------------------------------------------------------------
+  -- Multi Asset related tables.
+
+  MaTxOut
+    policy              ByteString          sqltype=hash28type
+    name                ByteString          sqltype=asset32type
+    quantity            DbWord64            sqltype=word64type
+    txOutId             TxOutId
+    UniqueMaTxOut       policy name txOutId
```

Comment:
Yes. Perfect.

Reply:
The fact that it is possible for a `(policy, name)` pair to exist in `MaTxOut` but not in `MaTxMint` concerns me.

Reply:
Also, why does one reference the `Tx` table and the other the `TxOut` table?

Reply:
The tx contains a value, the value minted.

The txout contains a value, the value of the output.